Editor/Helper/AddressablesImporterOdinInspectorDrawer.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace UnityAddressableImporter.Editor.Helper
+{
+    using System;
+    using System.Diagnostics;
+    using Object = UnityEngine.Object;
+
+    public class AddressablesImporterOdinInspectorDrawer : IDisposable
+    {
+        private Object _target;
+#if ODIN_INSPECTOR
+        private Sirenix.OdinInspector.Editor.PropertyTree _drawer;
+#endif
+
+        [Conditional("ODIN_INSPECTOR")]
+        public void Initialize(Object target)
+        {
+            _target = target;
+            _drawer = Sirenix.OdinInspector.Editor.PropertyTree.Create(_target);
```

Comment:
Smells wrong? reference _drawer, but not using #if macro? It seems the entire file should be wrapped with #if macro. The class called AddressablesImporter**Odin**InspectorDrawer after all. 

Please also test when Odin is not installed.